configurations/cadillac_srx_2013/docker-compose.yml

```diff
@@ -16,12 +16,26 @@
 version: '2'
 
 services:
+  roscore:
```

Comment:
What is the reason for taking roscore out of docker-compose-background?

Reply:
If we have it in the compose-background file, the rosmaster instance always stays up and we get one big rosout log. It's unclear how parameter loading/unloading also works in that instance, particularly if you're trying to tweak things. This is part of why we run into the new issue with the UI not being able to connect, since the roscore service now takes time to start up.

Reply:
Got you.